### ISSUE TITLE
Original Old Theme

### ISSUE THREAD

**Issue by Denizuga** [Yaro2709/New-Star#66]
Can the old classic theme be made with all the features?

**Comment by enesvtn** [comment]
Yes, it can be done, it is completely up to you, you need to develop something design-wise.
